astropy/io/fits/convenience.py

```diff
@@ -409,8 +409,16 @@ def writeto(filename, data, header=None, output_verify='exception',
         for more info.
 
     clobber : bool, optional
-        If `True`, and if filename already exists, it will overwrite
-        the file.  Default is `False`.
+        If `True`, overwrite the output file if exists.
```

Comment:
If we move `clobber` into `kwargs`, this text can go into `.. note::` section in the docstring. The code should also issue an explicit DeprecationWarning with this text.